docs/Theming.md

```diff
@@ -91,7 +91,7 @@ const VisitorFilter = props => {
     );
 };
 
-// The List component supports the `root`, `header`, `actions` and `noResults` CSS classes. Here we override the `header` and `actions` classes
```

Comment:
I don't understand this change. It's a space changed to a space. Can you explain?

Reply:
Yes, There was a zero with character detected by a plugin.
[Zero width Character locator](https://plugins.jetbrains.com/plugin/7448-zero-width-characters-locator/)